packages/@okta/vuepress-site/docs/concepts/feature-lifecycle-management/index.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Feature Lifecycle Management
+---
+
+# Feature Lifecycle Management
```

Comment:
I love the idea of this page. I think it would be much more useful if we used some sort of a diagrams to display state transitions of a given feature flag. Is that doable?

Reply:
Do we have such a diagram somewhere?